sd_webui_bayesian_merger/scorer.py

```diff
@@ -246,6 +246,8 @@ def open_image(self, image_path: Path) -> None:
         if system == "Windows":
             subprocess.run(["start", str(image_path)], shell=True, check=True)
         elif system == "Linux":
+            if 'microsoft-standard' in platform.uname().release:
+                print("Make sure to install xdg-open-wsl from here: https://github.com/cpbotha/xdg-open-wsl otherwise the images will NOT open.")
             subprocess.run(["xdg-open", str(image_path)], check=True)
```

Comment:
eog doesn't fully solve the issue since if a user is on windows 10, they will not have the ability to run graphical programs with WSL2. Just pointing the user to the xdg-open-wsl package is probably better than running into issues where users on windows 10 don't have it appear. Also, I added an update so that the message runs only once, since if the user has it we only want to alert them to the issue at the beginning.